src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx

```diff
@@ -100,6 +106,13 @@ const ArtworksGrid: React.FC<ArtworksGridProps> = ({
     setInitialFilterStateAction(filters)
   }, [])
 
+  useEffect(() => {
+    if (scrollToArtworksGrid) {
+      setTimeout(() => {
+        gridRef.current?.scrollToOffset({ offset: 0, animated: true })
+      }, 1000)
```

Comment:
question: What do you think about using `InteractionManager.runAfterInteractions` instead:

```suggestion
      InteractionManager.runAfterInteractions(() => {
        gridRef.current?.scrollToOffset({ offset: 0, animated: true })
      })
```

Reply:
I actually intentionally wanted to use a timeout here to make the experience slightly better and to navigate the users down only after seeing the artist image

Reply:
I always try to avoid timeouts because they lead to the order of events not being deterministic anymore.

It's not guaranteed that the image will be loaded after 1000 milliseconds, and scrolling to the artwork grid before the image is loaded seems to cause [this](https://artsy.slack.com/archives/C02BAQ5K7/p1699005594312359?thread_ts=1699003982.494799&cid=C02BAQ5K7) issue.


https://github.com/artsy/eigen/assets/4691889/8e1b23cc-ce4a-47be-8b62-c4197e31ce49

Reply:
Steps to reproduce:

1. Search for "Conrad Buff"
2. Click on Conrad Buff's "Artworks" tab